### DISCUSSION TITLE
Update the previous packages or install new package with older version to match?

### DISCUSSION THREAD

**Discussion by mariela-plaza** [plouc/nivo#2062] [Q&A]
Hello there!

I am currently working in an app which it's main job is to show a lot of data about genetics in the form of tables. My bosses now want to display this data in other types of visualizations too, so I researched and found the nivo library.

The thing is that the application already uses a nivo graph (the Chord graph, to be precise) in a different part of the application (the Home page). The versions of the nivo packages used for this visualitation are:

- "@nivo/chord": "^0.73.0"
- "@nivo/core": "^0.73.0"

My question now is, how do I hadle the new packages I want to install? Do I make them match the version we already have installed (0.73.0) or is it OK to upgrade the packages we are using to the latest version (0.79.1, at the time of writting this post)?

Thank you so much in advance! Have a great day!

**Comment by tkonopka** [comment]
Hi @mariela-plaza. There are pros and cons to both approaches (stick with a fixed version, or upgrade to latest). Personally, I would take this as an opportunity to upgrade all (nivo) packages to the latest versions. An important advantages is that you'd be able to use the documentation on nivo.rocks when developing the new visualizations, and not worry that certain features may not be available in your environment. Upgrading might cause the existing chord viz to break, but with luck such issues can be overcome with minor edits. If you run into specific trouble with the chord chart, please share a codesandbox, and we can investigate.

**Reply by mariela-plaza** [reply]
Hello @tkonopka, thank you for your response! I will take your advice and update our existing nivo packages to the latest version.

Have a great day, kind regards!
